Issue/61 add remove GitHub actions (#62)
HPI-SWA-Teaching/AutoTDD: packages/AutoTDD-GUI.package/ATDDGitHubActionResults.class/README.md

> ATDDTravisStatusResults shows the test results from the respective Travis Status.

Reviewer: ```suggestion
ATDDTravisStatusResults shows the test results of the selected Travis Status.
```
Respective can stay also, this is mostly about from vs. of
